package.json

```diff
@@ -17,6 +17,7 @@
     "affected:e2e": "nx affected:e2e",
     "affected:test": "nx affected:test",
     "affected:lint": "nx affected:lint",
+    "affected:lint:fix": "nx affected:lint --fix --base=origin/",
```

Comment:
I think the `base` should be set as default to `main` in `nx.json`. Then we won't need `--base` param in here

Reply:
fixed it and rebased